Source/Core/Core/DSP/Interpreter/DSPInterpreter.cpp

```diff
@@ -114,7 +116,7 @@ int Interpreter::RunCyclesDebug(int cycles)
       }
 
       // Idle skipping.
```

Comment:
Do we still need this (and the other) comment now that it is a fairly descriptive method?

Reply:
Nope, it's not necessary anymore. Removed it.